Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -739,39 +701,27 @@ namespace OpenInfraPlatform {
 				if(l_shape) {
 					if(l_shape->Depth && l_shape->Thickness) {
 						double h = l_shape->Depth * length_factor;
-						double b = h;
-
-						if(l_shape->Width) {
-							b = l_shape->Width * length_factor;
-						}
-
+						
+						double b = l_shape->Width.value_or(h);
+						

```

Comment:
A comment on the general style: no need for unnecessary line breaks. Usually, line breaks come as a new paragraph - so, if you are only initializing some variables, this is seen as a continuous paragraph, as the same sort of actions are done.

Reply:
Done